pom.xml

```diff
@@ -142,7 +155,7 @@
         <carbon.commons.version>4.8.7</carbon.commons.version>
         <carbon.commons.imp.pkg.version>[4.4.0, 5.0.0)</carbon.commons.imp.pkg.version>
         <!--Carbon identity version-->
-        <carbon.identity.framework.version>5.25.90</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.208</carbon.identity.framework.version>
```

Comment:
framework version needs to be updated after [https://github.com/wso2/carbon-identity-framework/pull/4674/files](https://github.com/wso2/carbon-identity-framework/pull/4674/files) is merged.